lib/berlim/guardian.ex

```diff
@@ -10,26 +10,17 @@ defmodule Berlim.Guardian do
     {:ok, to_string(user.id)}
   end
 
-  def resource_from_claims(%{"sub" => id, "type" => type}) do
-    case get_resource(id, type) do
-      nil ->
-        {:error, :resource_not_found}
-
-      user ->
-        {:ok, user}
-    end
+  def resource_from_claims(%{"sub" => id, "type" => "Admin"}) do
+    user = InternalAccounts.get_admin!(id)
```

Comment:
why don't you use like this {:ok, user} = InternalAccounts.get_admin!(id) ?

Reply:
rpz, acho que não rola esse match não
```
(MatchError) no match of right hand side value: %Berlim.InternalAccounts.Taxi ...
```